README.md

```diff
@@ -3,66 +3,71 @@
 [![License: UPL](https://img.shields.io/badge/license-UPL-green)](https://img.shields.io/badge/license-UPL-green) [![Quality gate](https://sonarcloud.io/api/project_badges/quality_gate?project=oracle-devrel_cd3-automation-toolkit)](https://sonarcloud.io/dashboard?id=oracle-devrel_cd3-automation-toolkit)
 
 ## Introduction
-CD3 stands for <b>C</b>loud <b>D</b>eployment <b>D</b>esign <b>D</b>eliverable.
+
+CD3 stands for **C**loud **D**eployment **D**esign **D**eliverable. 
+
 The CD3 Automation toolkit has been developed to help in automating the OCI resource object management. 
-<br><br>
```

Comment:
We need this section along with the image. We cant accept this change. Is there any particular reason you have modified this?